ui/src/layout/header.tsx

```diff
@@ -63,9 +69,30 @@ class Header extends React.Component<IHeaderProps, any> {
             >
               <MenuIcon />
             </IconButton>
+            <Typography
+              variant="title"
+              noWrap
+              style={{
+                marginRight: '40px'
+              }}>
+              <a href="/" className={"App-link"}><img src="/images/cloudcam-light.svg" className="App-logo" alt="logo" />
```

Comment:
svgs can be turned into react components https://www.smooth-code.com/open-source/svgr/